src/IRac.cpp

```diff
@@ -469,19 +469,27 @@ void IRac::amcor(IRAmcorAc *ac,
 /// @param[in] on The power setting.
 /// @param[in] mode The operation mode setting.
 /// @param[in] degrees The temperature setting in degrees.
+/// @param[in] sensorTemp The room (iFeel) temperature sensor reading in degrees
+///                       Celsius.
 /// @param[in] fan The speed setting for the fan.
 /// @param[in] swingv The vertical swing setting.
+/// @param[in] iFeel Whether to enable iFeel (remote temp) mode on the A/C unit.
 /// @param[in] turbo Run the device in turbo/powerful mode.
 /// @param[in] sleep Nr. of minutes for sleep mode.
 /// @note -1 is Off, >= 0 is on.
 void IRac::argo(IRArgoAC *ac,
                 const bool on, const stdAc::opmode_t mode, const float degrees,
-                const stdAc::fanspeed_t fan, const stdAc::swingv_t swingv,
+                const float sensorTemp, const stdAc::fanspeed_t fan,
+                const stdAc::swingv_t swingv, const bool iFeel,
                 const bool turbo, const int16_t sleep) {
   ac->begin();
   ac->setPower(on);
   ac->setMode(ac->convertMode(mode));
-  ac->setTemp(degrees);
+  ac->setTemp(static_cast<uint8_t>(degrees + 0.5));
+  if (sensorTemp != kNoTempValue) {
+    ac->setSensorTemp(static_cast<uint8_t>(sensorTemp + 0.5));
+  }
```

Comment:
See later comments about this. Not commenting here due to proximity to the other comment.